### PR TITLE
Localization support

### PR DESCRIPTION
**Pull request by ijaureguialzo**
Hello, I've just added some code to enable localized messages depending on the lang attribute of the html root node. This value is set by Sphinx based on the language property of the conf.py file.

You can see it working in test build builds:

https://lm-xml-apuntes.readthedocs.io/es/copybutton_test/apuntes/20_sintaxis_xml.html

It's a simplistic approach but I think it's enough for this case.

### CODE REVIEW

**Comment by choldgraf** [comment]
this is a great addition, I'd love to see more internationalization of the project!

It looks like in the circle site build, there aren't any copy buttons being added....any idea what's up?

Here's the built docs:

https://30-140029949-gh.circle-artifacts.com/0/html/index.html

and here's the error I saw:

```
Uncaught TypeError: Cannot read property 'copy' of undefined
    at clipboardButton (copybutton.js:37)
    at codeCells.forEach (copybutton.js:69)
    at NodeList.forEach (<anonymous>)
    at HTMLDocument.addCopyButtonToCodeCells (copybutton.js:66)
```

File: sphinx_copybutton/_static/copybutton.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
+// Localization support
+const locale = document.documentElement.lang || 'en'
```
Comment (choldgraf) [inline sphinx_copybutton/_static/copybutton.js]: one thought - it `documentElement.lang` exists but is not `en` or `es`, won't this fail? Can you add a check to make sure it defaults back to `en` if the dictionary doesn't have the language the person is requesting?

Comment (ijaureguialzo) [reply sphinx_copybutton/_static/copybutton.js]: You are right. I didn't take that into account.

Comment (ijaureguialzo) [reply sphinx_copybutton/_static/copybutton.js]: I think the check is better now but maybe I also missed something, it's late in this side of the world... :)

Comment (choldgraf) [reply sphinx_copybutton/_static/copybutton.js]: looks good to me! can keep an eye out for this if somebody notices disappearing copybuttons :-)  thanks for the patch!!

**Pull request merged**